tests/citeproc.rs

```diff
@@ -303,6 +310,39 @@ impl<'s> TestCaseBuilder<'s> {
     }
 }
 
+fn extract_from_html(html: &str) -> String {
```

Comment:
Could you implement bibliography test support on a separate PR? As is, it's harder to tell at a glance which changes were due to subsequent author substitution support and which changes were due to newly-supported bibliography tests.